.prettierrc.json

```diff
@@ -1,3 +1,7 @@
 {
-  "quoteProps": "consistent"
+  "quoteProps": "consistent",
+  "endOfLine": "lf",
+  "printWidth": 120,
+  "useTabs": true,
+  "trailingComma": "all"
```

Comment:
I like es5 personally - commas at the end of function params weirds me out

Reply:
sounds good (don't think this actually applies to anything yet in this code base)